File: javac/src/car.java
/**
 * @author Tyler
 */

public class car {
	
	// constructor
	public car(String carMake, int carYear) {
		this.make = carMake;
		this.year = carYear;
	}
	
	// public methods
	
	// getter
	public String getMake() {
		return this.make;
	}
	
	// setter
	public void setMake(String carMake) {
		this.make = carMake;
	}
	
	// getter
	public int getYear() {
		return this.year;
	}
	
	// setter
	public void setYear(int carYear) {
		this.year = carYear;
	}
	
	// private attributes
	private String make;
	private int year;
	
	// public attributes
	int array[] = new int[4];
	
	
}

File: cplusplus/making/Car.cpp
#include <string>
#include "Car.h"

Car::Car() {
    this->make = "unknown";
    this->model = "unknown";
    this->year = 0;
}

Car::Car(string make, string model, int year) {
    this->make = make;
    this->model = model;
    this->year = year;
}
